recipe/meta.yaml

```diff
@@ -27,10 +27,12 @@ test:
 
 about:
   home: https://poppler.freedesktop.org/
-  license: Adobe+GPLv2
+  license: GPL-2.0-or-later
```

Comment:
This package has 4 different licenses https://gitlab.freedesktop.org/poppler/poppler-data/-/blob/master/COPYING: MIT and GPL-2.0 and GPL-3.0 and Adobe (BSD-3-Clause)